bookmarks.html

```diff
@@ -14,99 +14,158 @@
     <title>Quizzzle</title>
   </head>
   <body>
-    <header class="zz-top">
+    <header class="zz-top title">
       <h1>Bookmarks</h1>
     </header>
 
     <main>
       <ul class="card-container" role="list">
-        <li class="card">
-          <div class="question-container">
-            <h2>Question</h2>
-            <p class="question">
+        <li class="card q-card">
```

Comment:
Better not go with abbreviations. It's a `card` or a `quiz-card` or `question-card`. `q-card` is not a good name.